### ISSUE TITLE
Empty install.sh file in version 0.22.0

### ISSUE THREAD

**Issue by Chever-John** [operator-framework/operator-lifecycle-manager#2841]
## Bug Report

In the latest version, there is an empty install.sh file, is there some unresolved issue?

**What did you do?**
I want to install OLM with the [script](https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/v0.22.0).

I found that there is a plant sh file of the latest version of OLM(0.22.0)

**Comment by alfieyfc** [comment]
https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.22.0/install.sh is 404.

Running directly with `scripts/install.sh v0.22.0` gives the following error. 
```
error: unable to read URL "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.22.0/crds.yaml", server reported 404 Not Found, status code=404
```

Waiting on the download directory to be updated.

**Comment by Chever-John** [comment]
@alfieyfc 
I am not the only one who has encountered this problem. I'm using a lower-level version now.

**Comment by imilchev** [comment]
Running into the same issue here as well

**Comment by timflannagan** [comment]
Yep, it looks like there's no install.sh or any of the requisite OLM YAML manifest files attached as release artifacts there. I can check whether there's an issue with the release pipeline where there's potential split brain behavior, but in the meantime, I can re-generate the release artifacts locally and update the release itself.

**Comment by timflannagan** [comment]
It would also be nice to have a github action that runs OLM's e2e testing suite after a release has been cut, so we can automatically detect this issues ahead of time. We have something similar to OLM's new rukpak component: https://github.com/operator-framework/rukpak that can be massaged into this repository.

**Comment by timflannagan** [comment]
Quick update: I ran `$ make release ver=v0.22.0 IMAGE_REPO=quay.io/operator-framework/olm` locally to generate the release artifacts, and then manually re-attached them to the v0.22.0 release. After doing that, I ran through the quickstart instructions on a kind cluster, and it looks like we're back in business.

**Comment by exdx** [comment]
Closing as @timflannagan resolved this issue. Please reopen if any more download issues occur.
